src/main/scala/nl/knaw/dans/lib/logging/servlet/package.scala

```diff
@@ -60,6 +64,12 @@ import org.scalatra.ActionResult
  * }}}
  *
  * ==Extension==
+ *
+ * // This example is a bridge too far.
```

Comment:
@jo-pol I added this section at your request, but apparently that's not what you intended?
From your description below I see you meant to add another example on how to mask only parts of the request/response. I added that in https://github.com/DANS-KNAW-jp/dans-scala-lib/pull/1, 970fa7e3dcc6fc83f5963ad1b1bcc99c4e9a02aa

Reply:
in other words, are you proposing to remove this whole section (starting from `==Extension==` until the end of this doc comment)
I would be in favor of that...